content/2018-10-31-xsimd.rst

```diff
@@ -0,0 +1,271 @@
+Bye bye boost.simd, welcome xsimd
+#################################
+
+:date: 2018-10-31
+:category: engineering
+:lang: en
+:authors: serge-sans-paille
+:summary: After five years of friendship, Pythran replaces ``boost.simd`` by
+          ``xsimd`` as its vectorization engine. This post goes through the
+          motivation of this change, and the advantage of this move.
+
+`boost.simd <https://github.com/NumScale/boost.simd>`_ provides a C++
+abstraction of vector type, allowing for efficient vectorization of array
+computations. It has been (optionally) used as part of the expression template
+engine of Pythran for a long time, a great collaboration that led to several
+patch in boost.simd, and great performance for Pythran.
+
+Unfortunately, the project has been silent over the last months (see for
+instance `this issue <https://github.com/NumScale/boost.simd/issues/546>`_ and
+I had to maintain a few custom patches. Turns out the project has been
+re-branded as **bSIMD** with a more restrictive license, as detailed in `another
+issue <https://github.com/NumScale/boost.simd/issues/545>`_. From the Pythran
+perspective, this is no good news.
```

Comment:
Technically, they have recently started talking about a freeware fork again (see the bottom of the linked issue). But practically, the way they have (mis)handled this killed my confidence in this project and made me question whether the NumScale economic model is sound at all. I guess same goes for you.